docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -260,8 +271,125 @@ Server can set age rating for current playback. Client can access this rating (r
     }
     ```
 
+## Managing playlists on server side
+
+To manage the media controller playlists in your server application, you must learn to create, save, delete playlists.
+
+1. Create a media controller playlist using the `createPlaylist()` method:
```

Comment:
Points 1 and 2 are in different maneer than the rest. Please use consistent form of the sentence in all of them.
1. **Create (verb)** **a mediacontroller playlist (noun)** .... 
2. **Adding (noun)** item to playlist **is available**
3. (and the rest) **To (do something)** ....

Reply:
Same comment as @pkosko

Reply:
Done